### ISSUE TITLE
Possibility to pass many parameters to cryptsetup

### ISSUE THREAD

**Issue by vaminakov** [agherzan/yubikey-full-disk-encryption#104]
Now line 147 of hook have quotes to each parameter:
`_tmp="$(printf %s "$_ykfde_passphrase" | cryptsetup luksOpen "$YKFDE_LUKS_DEV" "$YKFDE_LUKS_NAME" "$YKFDE_LUKS_OPTIONS" 2>&1)" `
So it can handle only one of them. If you have rare setup which need to pass more than one parameter, you'll not able to boot.
To fix this need just remove quotes:
`_tmp="$(printf %s "$_ykfde_passphrase" | cryptsetup luksOpen "$YKFDE_LUKS_DEV" "$YKFDE_LUKS_NAME" $YKFDE_LUKS_OPTIONS 2>&1)" `

**Comment by Vincent43** [comment]
Passing unquoted variable is something we want to avoid. Were the additional parameters about detached header? 

It's also possible to [store LUKS options in header](https://man.archlinux.org/man/cryptsetup-config.8)

**Comment by vaminakov** [comment]
@Vincent43 for example, in my configuration I'm using 
`--header=/header-luks.img --allow-discards`

**Comment by Vincent43** [comment]
Yeah so we had proposal for adding headers option [before](https://github.com/agherzan/yubikey-full-disk-encryption/pull/65).

The `cryptsetup config --header=/header-luks.img <device>` also should work
